EMMPlayer.cs

```diff
@@ -13,29 +14,52 @@ public class EMMPlayer : ModPlayer
 	{
 		public static EMMPlayer PlayerInfo(Player player) => player.GetModPlayer<EMMPlayer>();
 
-		public override void UpdateEquips(ref bool wallSpeedBuff, ref bool tileSpeedBuff, ref bool tileRangeBuff)
+		// Globals for modifiers
+		public bool holdingCursed;    // Whether currently holding a cursed item (take 1 damage per second)
```

Comment:
Same feedback as for EMMItem, however, there isn't anything generic right now for EMMPlayer. 
Feel free to make a new ModPlayer and name it ModifierPlayer, and leave EMMPlayer empty for now or remove it.